### PR TITLE
Add card image tag and horizontal card

### PR DESCRIPTION
**Pull request by scote**
✔️ Ready to merge

- [x] Card image
- [x] Card image floating FAB
- [x] Horizaton card

### CODE REVIEW

**Comment by scote** [comment]
That's weird on my machine I have several linter error when running `yarn lint:lib`. CircleCi does not seem to see those one.

**Comment by jfcere** [comment]
Can you give an example of those weird errors?

**Comment by scote** [comment]
I juste run the linter and I only get warnings for Renderer service this time.

File: demo/src/app/card/card.component.html [hunk 1]
```diff
@@ -94,14 +94,103 @@ <h5 class="light">Colors and hoverable</h5>
...
+</div>
+
 <div class="section">
 
   <h5 class="light">HTML Structure</h5>
 
 <app-code-snippet>
 &lt;mz-card
   class="blue-grey darken-1 white-text"
+  [horizontal]=="true"
```
Comment (scote) [inline demo/src/app/card/card.component.html]: Remove =

File: demo/src/app/card/card.component.ts [hunk 2]
```diff
@@ -12,6 +12,12 @@ import { IPropertyRow } from '../shared/properties-table/properties-table.compon
 })
 export class CardComponent {
   properties: IPropertyRow[] = [
+    { name: 'horizontal',
```
Comment (scote) [inline demo/src/app/card/card.component.ts]: Not align correctly

File: lib/src/card/card.component.ts [hunk 3]
```diff
@@ -39,8 +45,19 @@ export class MzCardComponent implements AfterViewInit {
     return this.isElementDisplayed(cardActionElement);
   }
 
+  private hasImageTagAndNotEmpty(): boolean {
+    const cardImagelement = this.cardImage.nativeElement.querySelector('mz-card-image');
+    return this.isElementDisplayed(cardImagelement);
+  }
+
+  private hasImageTitleTagAndNotEmpty(): boolean {
+
```
Comment (scote) [inline lib/src/card/card.component.ts]: Remove

**Pull request merged**